caesars-palace/blocks/product-details/product-details.js

```diff
@@ -59,7 +59,8 @@ export default async function decorate(block) {
       const rewardsEarn = overview['Caesars Rewards Earn'];
       const rewardsRedeem = overview['Caesars Rewards Redeem'];
       const rewardsLink = overview['Caesars Rewards Link'];
-
+      const superBowlDescription = overview['Super Bowl LVII'];
+      const superBowlLink = overview['Super Bowl LVII Link'];
```

Comment:
Instead of calling it specific to the content, let's make it a generic field because that extra column could be whatever they want in the future.
Something like `Additional Link`, `Additional Description` etc..